python/gui/qgsrasterformatsaveoptionswidget.sip

```diff
@@ -104,17 +104,17 @@ class QgsRasterFormatSaveOptionsWidget: QWidget
 %End
 
   private slots:
-    void on_mProfileNewButton_clicked() ;
+    void mProfileNewButton_clicked() ;
```

Comment:
Wow, that's a lot of `private slots` here :D

Reply:
Hmm... yes.... @3nids any idea why these are sneaking into the bindings?

Reply:
@nyalldawson should be fixed by #5328